data.js

```diff
@@ -330,7 +330,7 @@ const hardcodedData = {
     startingN: 1000,
     endingN: 9999999,
     resolution: 1000000,
-    graphFilename: null,
+    graphFilename: 'graph-images/countEvens.algo.png',
```

Comment:
We may want to consider a `path.resolve()` solution going forward. But, this is perfect for this iteration.